### PR TITLE
feat: redesign nlc & about page

### PR DESCRIPTION
**Pull request by codiini**
The empty space after the NLC states and faq links is meant to be the youtube embed. It didn't get captured in this screenshot for some reason.

Also, the NLC states have now increased to 40 but I made out a total of 39 since Connecticut is pending legislation early next year.

127 0 0 1_4000_nursinglicensurecompact html

127 0 0 1_4000_aboutrnmobility html

### CODE REVIEW

File: source/nursinglicensurecompact.html [hunk 1]
```diff
@@ -2,160 +2,149 @@
...
+    <img
+      class="col-lg-6 p-0"
+      src="images/nurses-arms-folded.webp"
+      alt="A group of five nurses, three males, two females, all standing with arms folded and smiling at the camera"
+    >
+    <!-- Image sourced from: Pexels, accessed on 8th Nov, 2024. No license or attribution required. -->
+
+    <div class="section-one__text bg-light col-lg-6 row flex-column m-0 pt-4 pt-lg-0 justify-content-center align-items-start">
+      <h3 style="font-weight: bold;" class="section-one__text--heading mb-4">
+        The Nursing Licensure Compact: The key to your mobility throughout 39 states
```
Comment (fulldecent) [inline source/nursinglicensurecompact.html:19]: calculate this value please

**Comment by fulldecent** [comment]
I see you have added a new data file to track this.

But instead could you please use the front matter inside each state's page to get this info? This will keep the site DRY.

Also please make sure the number "39 states" is calculated rather than literally typed.

All this means that if somebody updates one of the state pages to do:

```diff
  ---
  layout: page
  title: 'Alaska: RN license transfer, renewal & continuing education'
  description: "Learn about RN license transfer, renewal, and continuing education in Alaska. Get info to advance your 
  nursing practice."
  heading: 'Alaska'
  state: AK
  statename: Alaska
  published: true
  motto: 'Welcome to Alaska: The Last Frontier!'
+ compact: true
  ---
```

then all of our pages will update accordingly.

**Comment by codiini** [comment]
@fulldecent thanks for pointing those out. I've gotten rid of the data file in favor of the frontmatter in each page and also removed the hardcoded value

File: source/css/index.css [hunk 2]
```diff
@@ -183,6 +200,37 @@
...
+* About Page Section
+*/
+
+.about-section {
+  padding-top: 20px;
+}
+
+.about-section .about-section__text p {
+  width: 90%;
+  font-size: 18px;
```
Comment (fulldecent) [inline source/css/index.css:213]: Please see bootstrap's utilities for font sizes

https://getbootstrap.com/docs/5.0/getting-started/rfs/

tl;dr `rem` is preferred over `px`.

**Review by fulldecent — APPROVED** [review]

File: source/css/index.css (hunk 2 above)
Comment (codiini) [reply source/css/index.css:213]: @fulldecent using Bootstrap's RFS would mean setting up SCSS which I think you mentioned was a constraint.
I could manually just use the rem values and play around with css clamp as well if necessary.
